nav2_bt_navigator/behavior_trees/navigate_w_replanning_only_for_invalid_path.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+  This Behavior Tree replans the global path when it becomes invalid
+-->
+
+<root main_tree_to_execute="MainTree">
```

Comment:
It would be nice if this was setup with the recovery stuff as well (e.g. modify the current default BT and add in the changes for IsPathValid and save as a new file). That way out of the box someone gets the good nav BT stuff and this behavior at the same time versus a simplified demo BT